### DISCUSSION TITLE
lHTTP Metrics and TCP Metrics of Services in a namespace of inkerd have no traffic information

### DISCUSSION THREAD

**Discussion by guhaolin** [linkerd/linkerd2#11338] [Q&A] [answered]
There is information on HTTP Metrics and TCP Metrics in linkerd-viz > Services, but if you create a namespace yourself, there is no information on HTTP Metrics and TCP Metrics. I would like to ask how to set it up so that there is information?

k8s version: v1.22.2
linkerd version:  2.13.5 (stable)

**Comment by hawkw** [comment, marked as answer]
Are the services in that namespace meshed? See [the documentation](https://linkerd.io/2.14/tasks/adding-your-service/) for information on how to add your services to Linkerd.
